exercises/06-filter-data.md

```diff
@@ -0,0 +1,10 @@
+# Filter Data
+
+## 01.
+Return all the users that are female and in their 20's that have purchased more than 100 products.
```

Comment:
We can probably reword this to "Your goal is to return all female users in their 20's that have purchased more than 100 products. You will need to filter aggregate and row data."